politeiad/backendv2/tstorebe/plugins/comments/cmds.go

```diff
@@ -441,9 +441,18 @@ func (p *commentsPlugin) cmdNew(token []byte, payload string) (string, error) {
 		return "", err
 	}
 
+	// Ensure no extra data provided if not allowed
+	if !p.allowExtraData && (n.ExtraData != "" || n.ExtraDataHint != "") {
```

Comment:
Make this if statement a function since it copy/pasted several times.

Reply:
Good point will do